client/consul.go

```diff
@@ -336,3 +338,9 @@ func (c *ConsulService) makeCheck(service *structs.Service, check *structs.Servi
 	}
 	return cr
 }
+
+func (c *ConsulService) printLogMessage(message string, v ...interface{}) {
+	if _, ok := c.node.Attributes["consul.version"]; ok {
```

Comment:
I agree with @cbednarski on this one. I think we should skip the entire register loop if Consul was fingerprinted as missing. It doesn't make much sense otherwise to loop over every task, service, and check, and attempt a REST call we know will fail anyways.